src/main/java/org/jabref/gui/fieldeditors/FieldEditorFX.java

```diff
@@ -25,11 +25,13 @@ default boolean childIsFocused() {
 
     /**
      * Returns relative size of the field editor in terms of display space.
-     *
-     * A value of 1 means that the editor gets exactly as much space as all other regular editors.
-     * A value of 2 means that the editor gets twice as much space as regular editors.
+     * <p>
+     * A value of 1 means that the editor gets exactly as much space as all other regular editors. A value of 2 means
```

Comment:
Minor point: Did you do this manually or via your code editor? The previous version was better in my opinion.

Reply:
InteliJ autoformat. I added an additional <p>

It renders now as: (pressed <kbd>Ctrl</kbd>+<kbd>Q</kbd>)

![grafik](https://user-images.githubusercontent.com/1366654/81499460-1a45ff00-92cc-11ea-9243-a11a55840759.png)

Before, it rendered as 
![grafik](https://user-images.githubusercontent.com/1366654/81499471-2fbb2900-92cc-11ea-9ec0-bcc779750bff.png)